tests/integration/targets/meraki_ms_switchport/tasks/main.yml

```diff
@@ -724,3 +724,84 @@
     that:
       - idempotent_delete_mac.changed == False
       - idempotent_delete_mac.data is defined
+
+- name: Create access policy with auth_method is "my Radius Server"
```

Comment:
Integration tests are failing on this test.

```
TASK [meraki_ms_switchport : Create access policy with auth_method is "my Radius Server"] ***
fatal: [localhost]: FAILED! => {"changed": false, "msg": 404, "response": null, "status": 404, "url": "https://api.meraki.com/api/v1/networks/N_624874448297678702/switch/accessPolicies"}
```

I think it's because this network doesn't exist. Please convert these tests to use `org_name` and `net_name` so it's a bit more flexible. These tests are probably expecting a switch network ID to be pre-defined, which I don't necessarily include in the inventory.

Reply:
I fixed it ;)